### PR TITLE
Assignment_#3

### PR DESCRIPTION
**Pull request by mohammad-ayeshh**

### CODE REVIEW

File: src/ArrayOperations.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        for (int num : arr) {
+            if (num < min) {
+                min = num;
+            }
+        }
+        return min;
+    }
+    public static int getMax(int[] arr) {
+        //Method that returns the maximum value of integer array
+        int max = arr[0];
```
Comment (zurqan) [inline src/ArrayOperations.java:22]: you might need to check if the array is null or empty

File: src/ArrayOperations.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            }
+        }
+        return max;
+    }
+    public static double getAverage(int[] arr) {
+        //Method that returns the average of integer array values
+        if (arr.length == 0) {
+            return 0;
+        }
+        int sum = 0;
```
Comment (zurqan) [inline src/ArrayOperations.java]: you might need to define sum as double

File: src/ArrayOperations.java [hunk 3]
```diff
@@ -0,0 +1,75 @@
+public class ArrayOperations {
+    public static  boolean isNull(int[] arr){
+        return arr != null && arr.length != 0;
+    }
+    public static int getMin(int[] arr) {
+        //Method that returns the minimum value of integer array
+        if (isNull(arr)){
```
Comment (mohammad-ayeshh) [inline src/ArrayOperations.java:7]: check if the array is null or empty

File: src/ArrayOperations.java [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        }
+        return max;}
+        return -99999;
+    }
+    public static double getAverage(int[] arr) {
+        //Method that returns the average of integer array values
+        if (arr.length == 0) {
+            return 0;
+        }
+        double sum = 0;
```
Comment (mohammad-ayeshh) [inline src/ArrayOperations.java:36]: making the sum double

**Pull request merged**